### PR TITLE
Upgrading some nuget packages

### PR DESCRIPTION
**Pull request by raybooysen**
Upgraded:

NUnit
Autofac
Rx

### CODE REVIEW

**Comment by JakeGinnivan** [comment]
Can you upgrade Fody to latest pre-release too, this doesn't compile on vs2015

**Comment by raybooysen** [comment]
Sure.

**Comment by raybooysen** [comment]
Upgraded Fody

File: src/ReactiveTrader.sln [hunk 1]
```diff
@@ -36,6 +36,11 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Client", "Client", "{B05462
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Adaptive.ReactiveTrader.Client", "Adaptive.ReactiveTrader.Client\Adaptive.ReactiveTrader.Client.csproj", "{A894C01A-D326-40DF-BABC-0F807EB63A21}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{8D3B7107-069E-4DA5-91F7-FA4FCFD3AD6C}"
+	ProjectSection(SolutionItems) = preProject
+		.nuget\packages.config = .nuget\packages.config
```
Comment (rikoe) [inline src/ReactiveTrader.sln]: Why has a .nuget folder been added to the solution? Shouldn't be necessary?

Comment (raybooysen) [inline src/ReactiveTrader.sln]: The magic of letting nuget update your packages... ;)  I'll fix.

**Pull request merged**